projects/elife.yaml

```diff
@@ -404,8 +404,11 @@ api-gateway:
                     url: https://prod-elife-error-pages.s3.amazonaws.com/
                     codes:
                         404: "404.html"
-                        # temporarily disabled because of non-ASCII characters
-                        #503: "5xx.html"
+                        410: "410.html"
+                        503: "503.html"
+                    fallbacks:
```

Comment:
configuration format seems consistent

Reply:
This is `api-gateway`, but only the `test` stack. It can be used for testing in fact, but HTML error pages would not make sense on its other environments.